qiling/core.py

```diff
@@ -235,6 +239,9 @@ def run(self):
         runner = self.build_os_execution("runner")
         runner(self)
 
+        # Print syscalls and their call numbers.
+        self.dprint("Syscalls and number of invocations")
+        self.dprint(list(self.PE.syscall_count.items()))
```

Comment:
I guess we need to move this print to os/windows/x86.py and os/windows/x8664.py or check for the var exits before print

Reply:
Yep, my bad!